lib/matplotlib/collections.py

```diff
@@ -1254,6 +1255,139 @@ def set_verts_and_codes(self, verts, codes):
         self.stale = True
 
 
+class FillBetweenPolyCollection(PolyCollection):
+    def __init__(
+            self, ind_dir, ind, dep1, dep2=0, *,
```

Comment:
- naming: 53fd059074e976f534934afb3ca9796dea0a9d3c
- `dep2` should not have a default value: this is [the current default behaviour](https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.fill_between.html#matplotlib-pyplot-fill-between). Shouldn't we keep it?

Reply:
`fill_between` should keep it - one could discuss whether this is reasonable. Anyway, we won't change that for compatibility, and one can argue that it's a convenience on the high-level user-facing API.

In contrast. `FillBetweenPolyCollection` is a low-level building block and there the default does not make sense or is very helpful. I prefer to be explicit here (basically also holds for `_fill_between_x_or_y`, but it doesn't matter too much because that's private and we can change anytime we want).

Reply:
573b54d77d9094de36eccdb8010c94399a2ce60d